### PR TITLE
Add opportunities of client and update client detail

### PR DESCRIPTION
**Pull request by ndelarans**
image
image

### CODE REVIEW

File: src/features/client/pages/ClientDetail/ClientDetailMedium.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      getActions: () => {
+        return [
+          <Button
+            component={Link}
+            to={`#`}
+            variant="contained"
+            color="primary"
+          >
+            Seguimiento
+          </Button>,
```
Comment (dgop92) [inline src/features/client/pages/ClientDetail/ClientDetailMedium.tsx]: I think this should be normal button without a link

example:
https://github.com/exlusive-browser/exclusive-crm/blob/main/src/features/home/HomePage.tsx#L25-L27

**Review by dgop92 — CHANGES_REQUESTED** [review]
Another suggetion, add some padding to the table, 

image

**Review by dgop92 — APPROVED** [review]
ok

**Review by TheHoodini — APPROVED** [review]
Good

**Review by AlejandroSalas1143 — APPROVED** [review]
good

**Pull request merged**